README.md

```diff
@@ -62,6 +62,10 @@ First, acquire the source code by cloning the git repository:
 
     git clone git://github.com/JuliaLang/julia.git
 
+If you are not going to joing the development but only need the source for compilation, which means you don't need the commit history, add parameter `--depth 1` to shallow clone, it will really help you speed up the cloning process, and save the bandwidth and disk space:
```

Comment:
some types - "joing" should be "join," and no need for the article "the" in "save bandwidth and disk space."